examples/gallery/slider/LinearGradientSlider.cpp

```diff
@@ -0,0 +1,51 @@
+#include "LinearGradientSlider.h"
+
+#include <QskBoxBorderColors.h>
+#include <QskBoxBorderMetrics.h>
+
+LinearGradientSlider::LinearGradientSlider( QQuickItem* parent )
+    : LinearGradientSlider( Qt::Horizontal, parent )
+{
+}
+
+LinearGradientSlider::LinearGradientSlider( Qt::Orientation orientation, QQuickItem* parent )
+    : Inherited( orientation, parent )
+{
+    const QVector< QskGradientStop > gradientStop = {
+        { 0.0000, QColor::fromRgb( 255, 0, 0 ) },
+        { 0.1667, QColor::fromRgb( 255, 255, 0 ) },
+        { 0.3333, QColor::fromRgb( 0, 255, 0 ) },
+        { 0.5000, QColor::fromRgb( 0, 255, 255 ) },
+        { 0.6667, QColor::fromRgb( 0, 0, 255 ) },
+        { 0.8333, QColor::fromRgb( 255, 0, 255 ) },
+        { 1.0000, QColor::fromRgb( 255, 0, 0 ) },
+    };
+
```

Comment:
@uwerat since this control is 99% skinning maybe we shouldn't provide it as a Qsk control but as an example instead?